hack/header.tpl

```diff
@@ -1,6 +1,5 @@
 Copyright The Kubernetes NMState Authors.
 
```

Comment:
You have removed a line from the template, but in all the rendered headers there are two empty lines. Is it on purpose?

Reply:
Yes, it's on purpose. For some reason the goheader linter fails if there are 2 newlines in the template (most likely a bug). I tried to point this out in the commit message of 6c5d6b91a2d830085c87fde1f9f7e1fc78b407e2 , but maybe I was too brief.